source/robot_model/src/Model.cpp

```diff
@@ -1,22 +1,44 @@
 #include "robot_model/Model.hpp"
 #include "robot_model/Exceptions/FrameNotFoundException.hpp"
+#include "robot_model/Exceptions/InvalidJointStateSizeException.hpp"
 
 namespace RobotModel {
 Model::Model() : robot_name_(std::make_shared<StateRepresentation::Parameter<std::string>>("robot_name")),
                  urdf_path_(std::make_shared<StateRepresentation::Parameter<std::string>>("urdf_path")),
                  alpha_(std::make_shared<StateRepresentation::Parameter<double>>("alpha", 0.1)),
                  epsilon_(std::make_shared<StateRepresentation::Parameter<double>>("epsilon", 1e-2)),
-                 linear_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>("linear_velocity_limit", 2.0)),
-                 angular_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>("angular_velocity_limit", 100.0)),
-                 proportional_gain_(std::make_shared<StateRepresentation::Parameter<double>>("proportional_gain", 1.0)) {}
+                 linear_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>("linear_velocity_limit",
+                                                                                                 2.0)),
+                 angular_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>(
+                     "angular_velocity_limit",
+                     100.0)),
+                 proportional_gain_(std::make_shared<StateRepresentation::Parameter<double>>("proportional_gain",
+                                                                                             1.0)) {}
 
-Model::Model(const std::string& robot_name, const std::string& urdf_path) : robot_name_(std::make_shared<StateRepresentation::Parameter<std::string>>("robot_name", robot_name)),
-                                                                            urdf_path_(std::make_shared<StateRepresentation::Parameter<std::string>>("urdf_path", urdf_path)),
-                                                                            alpha_(std::make_shared<StateRepresentation::Parameter<double>>("alpha", 0.1)),
-                                                                            epsilon_(std::make_shared<StateRepresentation::Parameter<double>>("epsilon", 1e-2)),
-                                                                            linear_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>("linear_velocity_limit", 2.0)),
-                                                                            angular_velocity_limit_(std::make_shared<StateRepresentation::Parameter<double>>("angular_velocity_limit", 100.0)),
-                                                                            proportional_gain_(std::make_shared<StateRepresentation::Parameter<double>>("proportional_gain", 1.0)) {
+Model::Model(const std::string& robot_name, const std::string& urdf_path) : robot_name_(std::make_shared<
```

Comment:
I think we may have to tweak some format file settings so that CLion doesn't garble these initialisers 🤔 It's doing the best it can. In my previous job the style rule for constructors was

```c++
Foo::Foo(.., ..) :
    member1(val),
    member2(val) {
    ...
}
```
(no continuation indent)

Reply:
Yep we need to get clear with the formatting because we have several issues now within this PR